projects/sandbox/deploy/deploy/trigger.py

```diff
@@ -0,0 +1,157 @@
+import json
+import logging
+import time
+from dataclasses import asdict, dataclass
+from pathlib import Path
+from typing import Literal, Optional, Union
+
+import numpy as np
+from ligo.gracedb.rest import GraceDb
+
+from bbhnet.analysis.ledger.events import TimeSlideEventSet
+
+Gdb = Literal["playground", "test", "production"]
+
+
+@dataclass
+class Event:
```

Comment:
I think we should also report something like `IFOs: ["H1", "L1"]`

Reply:
This indicates which IFOs were used to sense the trigger I guess?

Reply:
Exactly, yeah. If we ever had a single (or triple, or quadruple...) running mode, we could report whichever interferometers were used to make the detection